src/aliased_buffer.h

```diff
@@ -32,6 +32,31 @@ template <class NativeT,
           typename = std::enable_if_t<std::is_scalar<NativeT>::value>>
 class AliasedBufferBase {
  public:
+  /**
```

Comment:
is this being PRed in a sepearte #32778 ?

Reply:
This is being refactored out, actually. The need to use v8::BackingStore for this means this new constructor is no longer needed. AliasedStruct will also replace several of the uses of AliasedBuffer throughout.